scripts/publish-process/update-docs-deps.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# Backpack - Skyscanner's Design System
+#
+# Copyright 2016-2021 Skyscanner Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BRANCH_NAME="autobump-upgrade-bpk-dependencies"
+
+echo "--- Cloning Backpack docs ---"
+git clone git@github.com:skyscanner/backpack-docs.git
+
+cd backpack-docs
+
+echo "--- Checkout upgrade branch ---"
+git checkout -b $BRANCH_NAME
+
+echo "--- Updating bpk dependencies ---"
+npx npm-check-updates "/^bpk\\-(.*)$/" -u && npm i
```

Comment:
I feel like maybe the backpack-docs repo should have it's own script for bumping which this script could just use? But maybe that's over engineering it?

Reply:
Yeah - I don't think we need to over engineer here, this is just a simple script to help out, as the docs site would never auto run a script.

I just translated my workflow into a script here for convience and having steps across both repos adds to the maintenace burden which we want to avoid

Reply:
I do also however see we have the npx part as a npm command we can invoke so could just use that instead

https://github.com/Skyscanner/backpack-docs/blob/main/package.json#L45

Reply:
Yeah. I guess maybe that should do the `npm i` as well as it hasn't technically upgraded them until that's done

Reply:
Docs site script has been fixed to include this and now the step has been simplified! Thanks for the suggestion looks cleaner now!